pydicom/filewriter.py

```diff
@@ -284,7 +291,7 @@ def write_DA(fp, data_element, padding=' '):
     if isinstance(val, (str, compat.string_types)):
         write_string(fp, data_element, padding)
     else:
-        if isinstance(val, (list, tuple)):
+        if is_multi_value(val):
```

Comment:
@massich: I prefer to separate unrelated refactorings from the bug fix. But I will make `is_multi_value` private.